### DISCUSSION TITLE
Using custom peak matrix to identify doublets

### DISCUSSION THREAD

**Discussion by ScottyZhang-wh** [GreenleafLab/ArchR#813] [Questions / Documentation]
Hi. Currently I have a custom peak matrix, however, I have no access to the raw sequencing data like .bed or .fragment file. I intend to identify the doublets through my custom peak matrix. How can I reach the goal. I have tried to calculate the doublet scores based on the imported peak matrix, but it doesn't work. It seems that all downstream analyses are based on the Arrowfile. But how can I create an arrowfile only with the peak matrix/ tilematrix? Is there any other way that can help me filter the doublets with only the matrix? Thank you so much!

**Comment by rcorces** [comment]
ArchR is not able to start from a peak matrix and we dont have any near-term plans to enable this.
